packages/react/src/components/__tests__/OrganizationProfile.test.tsx

```diff
@@ -19,18 +19,6 @@ describe('<OrganizationProfile/>', () => {
       }).toMatchTypeOf<OrganizationProfileComponentProps>();
     });
 
-    test('when path is filled, routing must only have path as value', () => {
-      expectTypeOf({
-        path: '/org',
-        routing: 'virtual' as const,
-      }).not.toMatchTypeOf<OrganizationProfileComponentProps>();
-
-      expectTypeOf({
-        path: '/org',
-        routing: 'hash' as const,
-      }).not.toMatchTypeOf<OrganizationProfileComponentProps>();
-    });
```

Comment:
Removing as the types do not enforce this now and relying on https://github.com/clerk/javascript/pull/4977/files#r1925959972